src/main/kotlin/no/nav/familie/ef/sak/vedtak/VedtakService.kt

```diff
@@ -100,6 +119,62 @@ class VedtakService(
     fun hentHarAktivtVedtak(behandlingId: UUID, localDate: LocalDate = LocalDate.now()): Boolean {
         return hentVedtak(behandlingId).erVedtakAktivtForDato(localDate)
     }
+
+    @Transactional
+    fun angreSendTilBeslutter(saksbehandling: Saksbehandling) {
+        val vedtak = hentVedtak(behandlingId = saksbehandling.id)
+
+        validerKanAngreSendTilBeslutter(saksbehandling, vedtak)
+
+        behandlingService.oppdaterStegPåBehandling(saksbehandling.id, steg = StegType.SEND_TIL_BESLUTTER)
+        behandlingService.oppdaterStatusPåBehandling(saksbehandling.id, status = BehandlingStatus.UTREDES)
+        behandlingshistorikkService.opprettHistorikkInnslag(
+            behandlingId = saksbehandling.id,
+            stegtype = saksbehandling.steg,
```

Comment:
Her bruker vi saksbehandlingen sitt steg, og ikke det oppdaterte steger, for å ikke skape forvirring/ev feil så tror jeg det kan være bra å flytte 
`oppdaterStegPåBehandling` og `oppdaterStatusPåBehandling` til etter at vi oppretter historikkinslag 👀 
I `håndterSteg` gjøes det også slik (oppdatering etter `opprettHistorikkInslag`)
Føles nesten som at dette hadde kunnet vært noe som gjøres som et steg (i `BeslutteVedtakSteg`eller en egen StegService som også er for BESLUTTE_VEDTAK), då får man også validering på at det gjøres på "riktig steg", og koden blir mer "isolert"

Reply:
Fint innspill 👏  Vi lar nå stegservice håndtere og validering og oppdatering av steg.